tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -16,11 +16,19 @@
 
 #include <nnstreamer_plugin_api_filter.h>
 
+#if defined(__aarch64__)
+#define ARCH "aarch64"
+#elif defined(__arm__)
+#define ARCH "arm"
+#else /* x86_64 */
+#define ARCH "x86_64"
+#endif
```

Comment:
You need a default value for ```ARCH``` here. Is it possible to build it without ```gbs```?

Reply:
```__aarch__``` and ```__arm__``` are not defined by gbs. They are defined by compilers: https://www.keil.com/support/man/docs/armclang_ref/armclang_ref_chr1383660321827.htm

Reply:
Anyway, please provide default behavior as @wooksong said. (e.g., "invalid")

Reply:
Absolutely